lib/WTSI/NPG/HTS/PacBio/Sequel/AnalysisPublisher.pm

```diff
@@ -54,6 +57,13 @@ has 'analysis_path' =>
    required      => 1,
    documentation => 'PacBio root analysis job path');
 
+has 'is_oninstrument' =>
+  (isa           => 'Bool',
+   is            => 'ro',
+   required      => 0,
+   default       => 0,
+   documentation => 'The analysis was done on the instrument (not in SMRT Link)');
```

Comment:
Could you add a little extra info about why this means  `$subdironly` in practice i.e. where it is being passed into methods as a param with that name.